tests/backend/test_postgres_db.py

```diff
@@ -0,0 +1,62 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import re
+
+import pytest
+
+from parsec.backend.config import PostgreSQLBlockStoreConfig
+
+
+@pytest.mark.trio
+@pytest.mark.postgresql
+async def test_postgresql_connection_ok(
+    postgresql_url, alice, bob, backend_factory, backend_sock_factory
+):
+    async with backend_factory(
+        config={"blockstore_config": PostgreSQLBlockStoreConfig(), "db_url": postgresql_url}
```

Comment:
why `"blockstore_config": PostgreSQLBlockStoreConfig()` ? I would think `blockstore` fixtures returns `PostgreSQLBlockStoreConfig()` for postgresql tests (and this test is marked as postgresql only)

Reply:
Does work, but is not even needed actually